src/main/java/uk/gov/companieshouse/api/accounts/service/AbstractService.java

```diff
@@ -1,24 +1,28 @@
 package uk.gov.companieshouse.api.accounts.service;
 
-import java.security.NoSuchAlgorithmException;
 import uk.gov.companieshouse.api.accounts.exception.DataException;
 import uk.gov.companieshouse.api.accounts.model.entity.BaseEntity;
 import uk.gov.companieshouse.api.accounts.model.rest.RestObject;
 import uk.gov.companieshouse.api.accounts.service.response.ResponseObject;
 import uk.gov.companieshouse.api.accounts.transaction.Transaction;
 
-public interface AbstractService<T extends RestObject, U extends BaseEntity> {
+public interface AbstractService<T extends RestObject, U extends BaseEntity, V extends BaseEntity> {
```

Comment:
only the first generic type is used in this interface, if the others are not required at this level they shouldn't be part of the interface interface spec